main.mk

```
@@ -55,6 +55,8 @@ endif
 %.xml: %.org
 	$(oxtradoc) -m outline-to-xml -n "$@" $< > $@
 
+XML2RFCV3OPTION := $(if $(V3),--v3)
```

I've tended to use the following form:

```suggestion
ifeq (true,$(XML2RFCV3))
XML2RFCV3 := --v3
endif
```

It's more verbose, but it makes the check - and the variable - more explicit.